### DISCUSSION TITLE
Ability to explicitly specify CefSharpPlatformTarget if we want x64 only when built in AnyCPU

### DISCUSSION THREAD

**Discussion by yefka** [cefsharp/CefSharp#3971] [Q&A] [answered]
Hi,

My application is built as AnyCPU, but we are using only x64 version of CefSharp.
This application is deployed as a Clickonce, so I've added this to my csproj : 
<CefSharpBuildAction>Content</CefSharpBuildAction>

Is there any way to disable AnyCPU support and get only x64 binaries in the output ?
Actually as a workaround I've explicitly excluded the x86 binaries with (for each file) : 
```
    <PublishFile Include="x86\CefSharp.BrowserSubprocess.Core.dll">
      <Visible>False</Visible>
      <Group>
      </Group>
      <TargetPath>
      </TargetPath>
      <PublishState>**Exclude**</PublishState>
      <IncludeHash>False</IncludeHash>
      <FileType>File</FileType>
    </PublishFile>

```

Is there any better way to acheive this ?
I would like to explicitly define CefSharpPlatformTarget, but this value is explicitly set in to <CefSharpPlatformTarget>$(PlatformTarget)</CefSharpPlatformTarget>
https://github.com/cefsharp/CefSharp/blob/dc9d0a2cca5c414c94805ea9c8ab9a34675cae5d/NuGet/CefSharp.Common.targets#L121

Thank you.

**Comment by amaitland** [comment, marked as answer]
Use [CefSharpPlatformTargetOverride](https://github.com/cefsharp/CefSharp/blob/dc9d0a2cca5c414c94805ea9c8ab9a34675cae5d/NuGet/CefSharp.Common.targets#L125) set to `x64`

**Reply by yefka** [reply]
Thank you so much, that's perfect.
